thinc/shims/pytorch.py

```diff
@@ -40,6 +48,8 @@ def __init__(
         mixed_precision: bool = False,
         grad_scaler: Optional[PyTorchGradScaler] = None,
         device: Optional["torch.device"] = None,
+        serialize_model: Optional[Callable[[Any], bytes]] = None,
```

Comment:
Can't we pass `default_(de)?serialize` models as default values for these arguments?

Reply:
I just wanted to keep the parameters in line with [how it's done in `PyTorchWrapper`](https://github.dev/explosion/thinc/blob/e4a11573fdc6ee3c6c0f03641d4dd2407ac4f9bd/thinc/layers/pytorchwrapper.py#L76). This also lets us define the default methods anywhere in the module (as opposed to above the class for visibility when using them as default arguments).

Reply:
I agree that default args in both places would be nicer. If in doubt, I'd favor consistency.

Reply:
So, after some discussion, we decided to leave this as-is.